src/angular/form-field/BUILD.bazel

```diff
@@ -36,30 +33,6 @@ sass_binary(
     ],
 )
 
-ng_test_library(
```

Comment:
Form field doesn't have a test anymore?

Reply:
Yes, because it is being tested in the input module.